samples/apps/tinyRA/README.md

```diff
@@ -0,0 +1,33 @@
+# TinyRA
+
+TinyRA is a minimalistic Research Assistant built with AutoGen agents.
+
+## Installation
+
+To install TinyRA, clone the repository and install it using pip:
```

Comment:
Do we want to create a Pypi package for it? It would be helpful for majority users, who don't want to touch the source code (also good for tutorial).

Reply:
I have one already! :)

Reply:
I will make a release for you right now.